pkg/generate/app/cmd/newapp.go

```diff
@@ -1071,6 +1070,7 @@ func (c *AppConfig) run(acceptors app.Acceptors) (*AppResult, error) {
 		}, nil
 	}
 
+	imageRefs := components.ImageComponentRefs()
 	pipelines, err := c.buildPipelines(imageRefs, env)
```

Comment:
yup, didn't notice that was the only remaining reference to that variable now.  fixed.